### ISSUE TITLE
Login does not work for http://nraynaud.github.io/webgcode/webapp/visucamTest.html#/

### ISSUE THREAD

**Issue by ansgar75** [nraynaud/webgcode#22]
Hi,

I tried to open your link http://nraynaud.github.io/webgcode/webapp/visucamTest.html# but the login does not work so far.
i tried this in Firefox as well as Chrome.

Thank you

**Comment by nraynaud** [comment]
Yes, thanks. Firebase sent me an email that I ignored. I will look into it.

**Comment by dirkse** [comment]
Same problem? But look like some nice work so far...
error |  
-- | --
code | "SERVICE_UNAVAILABLE"
statusCode | 410
message | "SDK Update Required: See https://goo.gl/2cKQWm."
details | "SDK Update Required: See https://goo.gl/2cKQWm."
